musicdiff/__init__.py

```diff
@@ -81,8 +81,9 @@ def diff(score1: Union[str, Path, m21.stream.Score], # can be file or Score
         if not badArg1:
             try:
                 score1 = m21.converter.parse(score1, forceSource = force_parse)
-            except:
+            except Exception as e:
                 print(f'score1 ({fileName1}) could not be parsed by music21', file=sys.stderr)
+                print(e)
```

Comment:
print(e, file=sys.stderr)
... so that apps whose primary output is stdout don't get these errors intermingled.